extensions/data-transfer/portability-data-transfer-google/src/test/java/org/datatransferproject/datatransfer/google/mediaModels/GoogleMediaItemTest.java

```diff
@@ -135,6 +135,8 @@ public void getMimeType_videoModel_nullMimeTypeReturnsDefault() {
 
     // Default defined in GoogleMediaItem
     assertEquals("video/mp4", videoModel.getMimeType());
+    assertEquals("Mon Oct 02 22:33:38 UTC 2023", videoModel.getUploadedTime().toString());
```

Comment:
lets set this in the test

Reply:
Created separate tests to test upload Time